galata/test/documentation/debugger.test.ts

```diff
@@ -76,19 +76,21 @@ test.describe('Debugger', () => {
 
     await createNotebook(page);
 
-    const runButton = await page.waitForSelector(
-      '.jp-Toolbar-item >> [data-command="notebook:run-cell-and-select-next"]'
-    );
+    const runButton = await page
+      .locator('.jp-Toolbar-item')
+      .locator('[data-command="notebook:run-cell-and-select-next"]')
+      .getByRole('button')
+      .elementHandle();
```

Comment:
If the purpose of this change is to remove deprecated `waitForSelector`, we should probably avoid `elementHandle` which is also deprecated.

```suggestion
    const runButton = page.locator(
      '.jp-Toolbar-item >> [data-command="notebook:run-cell-and-select-next"]'
    );
    await runButton.waitFor();
```

Reply:
Indeed it is. But we still need the elementHandle to position the emulated mouse pointer. Hence the current proposal.